### PR TITLE
ECPU update for ADB

### PR DESCRIPTION
**Pull request by martygubar**
Updated building block for provisioning ADB to use ECPUs

### CODE REVIEW

**Review by klazarz — APPROVED** [review]

**Review by klazarz — DISMISSED** [review]

**Comment by klazarz** [comment]
@martygubar can u please check the conflicts?

File: building-blocks/tasks/adb/images/adb-create-screen-configure-db.png
Comment (martygubar) [inline building-blocks/tasks/adb/images/adb-create-screen-configure-db.png:1]: This new image is correct. It shows the new ECPUs

File: building-blocks/tasks/adb/provision-body.md
Comment (martygubar) [inline building-blocks/tasks/adb/provision-body.md:1]: The content now shows ECPUs instead of OCPUs. The new one is correct

**Review by martygubar — COMMENTED** [review]
Completed ECPU update

**Review by davidastart — APPROVED** [review]

**Pull request merged**